certified-connectors/Tesseron ASM Ticket/apiDefinition.swagger.json

```diff
@@ -0,0 +1,819 @@
+                  "title": "Ticket Number"
+                },
+                "TicketType": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "ID of the ticket type",
+                  "title": "Ticket Type ID"
+                },
+                "TicketTypeName": {
+                  "type": "string",
+                  "description": "Name of the ticket type",
+                  "title": "Ticket Type Name"
+                },
+                "TicketHeader": {
+                  "type": "string",
+                  "description": "Subject of the ticket",
+                  "title": "Ticket Header"
+                },
+                "TicketText": {
+                  "type": "string",
+                  "description": "Description of the ticket",
+                  "title": "Ticket Text"
+                },
+                "ReferenceNumber": {
+                  "type": "string",
+                  "description": "Reference Number of the ticket",
+                  "title": "Reference Number"
+                },
+                "EntranceType": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "ID of the ticket entrance type",
+                  "title": "Entrance Type ID"
+                },
+                "EntranceTypeName": {
+                  "type": "string",
+                  "description": "Name of the ticket entrance type",
+                  "title": "Entrance Type Name"
+                },
+                "AreaId": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "ID of the ticket area",
+                  "title": "Area ID"
+                },
+                "AreaName": {
+                  "type": "string",
+                  "description": "Name of the ticket area",
+                  "title": "Area Name"
+                },
+                "ProjectId": {
+                  "type": "string",
+                  "description": "ID of the related project",
+                  "title": "Project ID"
+                },
+                "ProjectPhaseId": {
+                  "type": "string",
+                  "description": "Phase of the related project",
+                  "title": "Project Phase ID"
+                },
+                "ProjectPhaseTaskId": {
+                  "type": "string",
+                  "description": "Task of the related project",
+                  "title": "Project Phase Task ID"
+                },
+                "Tags": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "title": "Tags"
+                  },
+                  "description": "Tags"
+                },
+                "ReleasedOption": {
+                  "type": "boolean",
+                  "description": "ReleasedOption",
```

Comment:
@developmentLL please look through descriptions and make sure they meet [English strings standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).